upstream-community-operators/percona-server-mongodb-operator/1.5.0/percona-server-mongodb-operator.v1.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,362 @@
+
+
+    * **Add Monitoring** - [Percona Monitoring and Management (PMM) can be
+    easily
+    deployed](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/monitoring.html)
+    to monitor your Percona Server for MongoDB replica set(s). The recommended
+    installation process uses Helm, the package manager for Kubernetes.
+
+
+    * **Automate Your Backups** - [configure automatic
+    backups](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/backups.html)
+    to run on a scheduled basis or run an on-demand backup at any time. Backups
+    are performed using Percona Backup for MongoDB (PBM) and can be stored on
+    local PVs or in any S3-compatible cloud storage provider.
+
+
+    ### Common Configurations
+
+
+    * **Set Member as Arbiter** - [Set up a replica set which contains an
+    arbiter](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/arbiter.html),
+    which participates in elections but does not store any data.  This reduces
+    storage costs while helping maintain replica set integrity.
+
+
+    * **Expose Members Outside K8S** - [by appropriately configuring the
+    ServiceType](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/expose.html)
+    you can expose replica set members outside of Kubernetes or provide
+    statically assigned IP addresses.
+
+
+    * **Utilize Local Storage Options** - [with support for Local Storage you
+    can mount existing data
+    directories](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/storage.html)
+    into your replica set managed by Kubernetes or utilize high performance
+    hardware for local storage rather than network storage for your database.
+
+
+    ### Before You Start
+
+
+    Add the PSMDB user `Secret` to Kubernetes. User information must be placed
+    in the data section of the `secrets.yaml`
+
+    file with Base64-encoded logins and passwords for the user accounts.
+
+
+    Below is a sample `secrets.yaml` file for the correct formatting.
+
+
+    ```
+
+    apiVersion: v1
+
+    kind: Secret
+
+    metadata:
+      name: my-cluster-name-secrets
+    type: Opaque
+
+    data:
+      MONGODB_BACKUP_USER: YmFja3Vw
+      MONGODB_BACKUP_PASSWORD: YmFja3VwMTIzNDU2
+      MONGODB_CLUSTER_ADMIN_USER: Y2x1c3RlckFkbWlu
+      MONGODB_CLUSTER_ADMIN_PASSWORD: Y2x1c3RlckFkbWluMTIzNDU2
+      MONGODB_CLUSTER_MONITOR_USER: Y2x1c3Rlck1vbml0b3I=
+      MONGODB_CLUSTER_MONITOR_PASSWORD: Y2x1c3Rlck1vbml0b3IxMjM0NTY=
+      MONGODB_USER_ADMIN_USER: dXNlckFkbWlu
+      MONGODB_USER_ADMIN_PASSWORD: dXNlckFkbWluMTIzNDU2
+      PMM_SERVER_USER: cG1t
+      PMM_SERVER_PASSWORD: c3VwYXxefHBheno=
+    ```
+
+
+
+    *  Amazon Elastic Container Service for Kubernetes (EKS) was added to the
+    list of the officially supported platforms
+
+    *  Percona Server for MongoDB 4.2 is now supported
+
+    *  OpenShift Container Platform 4.5 is now supported
+
+
+
+
+  maturity: stable
+  version: 1.5.0
+  replaces: ''
```

Comment:
@tplavcic i you want to start new channel, please modify pakckage.yaml file and remove `replaces` here

Reply:
@mvalahtv I have put a previous version into replaces now. Please check if it makes sense now. Thanks!